challenges/objects-arrays.js

```diff
@@ -99,23 +156,40 @@ The zoos need a list of all their animal's names (animal_name only) converted to
 
 */
 
-const lowCaseAnimalNames = [];
+
+const lowCaseAnimalNames = zooAnimals.map(animal => {
+
+  return animal.animal_name.toLocaleLowerCase();
+  
+});
+
+
 console.log(lowCaseAnimalNames);
 
 /* Request 3: .filter() 
 
 The zoos are concerned about animals with a lower population count. Using filter, create a new array of objects called lowPopulationAnimals which contains only the animals with a population less than 5.
 
 */
-const lowPopulationAnimals = [];
+
+
+const lowPopulationAnimals = zooAnimals.filter(animal => {
+return animal.population < 5;
+});
+
+
+
 console.log(lowPopulationAnimals);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States. Find the total population from all the zoos using the .reduce() method. Remember the reduce method takes two arguments: a callback (which itself takes two args), and an initial value for the count.
 
 */
-const populationTotal = 0;
+
+const populationTotal = zooAnimals.map(animal => animal.population).reduce((populationTotal, animal_pop)=>populationTotal + animal_pop);
```

Comment:
Perfect. Make sure to spread long code like this into multple lines when possible to increase readability 🙂